src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -8,12 +8,47 @@
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_BOOKING = new Prefix("b/");
     public static final Prefix PREFIX_RESIDENCE_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_CLEAN_STATUS_TAG = new Prefix("c/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_BOOKING_START_DATE = new Prefix("s/");
     public static final Prefix PREFIX_BOOKING_END_DATE = new Prefix("e/");
     public static final Prefix PREFIX_RESIDENCE = new Prefix("r/");
+
+    public static Prefix getPrefixName() {
+        return PREFIX_NAME;
+    }
+
+    public static Prefix getPrefixPhone() {
+        return PREFIX_PHONE;
+    }
+
```

Comment:
Hey, I think you could use these Prefixes directly by importing them without the need for these getters! Since they're public static similar to how we see it used it other command classes :)